Sorter.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
```

Comment:
No need to have ascending and descending methods. You can have one and the other will be inverse of it